.storybook/global.less

```diff
@@ -0,0 +1,42 @@
+/**
+ * This file contains styles that are used in prod that ultimately
+ * get passed onto Perseus components. This file is imported in the
+ * .storybook/preview.js file so that the components can have the same
+ * styles as prod when viewed within Storybook.
+ */
+
+// Variables
+@offBlack: #21242c;
+@wonderBlocksFontFamily: "Lato", "Noto Sans", "Helvetica", "Corbel", sans-serif;
+
+// Layers
+@layer reset, shared;
+
+@layer reset {
+    // A CSS reset. Needed for every page.
+    @import (inline) "./reset.css";
+}
+
+@layer shared {
+    html,
+    body {
+        height: 100%;
+    }
+
+    body {
+        font-family: @wonderBlocksFontFamily, "Helvetica", "Corbel", sans-serif;
+        font-size: 14px;
+        margin: 0;
+        color: @offBlack;
+        line-height: 1.4;
+        min-width: 0 !important;
```

Comment:
Does this really need `!important`?

Reply:
Ah, that was already there in webapp with the comment

```
/* Needs !important because we might load shared.css later */
```

But I don't think that's relevant anymore? I don't see a `shared.css` anywhere. I'll remove the `!important`.